### PR TITLE
e2e: Monitoring operator has one pod since OCP 4.13

### PR DESCRIPTION
**Pull request by jsliacan**
Starting with OpenShift 4.13, monitoring operator will have 1 pod. We used to check for `2/2`. 

 ```xml 
<failure message="Step stdout matches ".*cluster-monitoring-operator-\w+-\w+\ *2/2\ *Running.*": output did not match. Expected: '.*cluster-monitoring-operator-\w+-\w+\ *2/2\ *Running.*', Actual: 'NAME READY STATUS RESTARTS AGE alertmanager-main-0 6/6 Running 1 (3m58s ago) 4m5s cluster-monitoring-operator-78b755f457-6k4df 1/1 Running 0 4m37s kube-state-metrics-58957c56bc-v268k 3/3 Running 0 4m11s node-exporter-4rx4t 2/2 Running 0 4m10s openshift-state-metrics-5449996978-p585x 3/3 Running 0 4m11s prometheus-adapter-7f65dfd454-jrmqt 1/1 Running 0 4m8s prometheus-k8s-0 6/6 Running 0 4m1s prometheus-operator-6c5b488ff4-vgq9w 2/2 Running 0 4m20s prometheus-operator-admission-webhook-c8c577b57-bzww2 1/1 Running 0 4m28s telemeter-client-dcdcf886d-2hb9b 3/3 Running 0 4m5s thanos-querier-57c756949c-pz82d 6/6 Running 0 4m3s'"/>
```
Should only be merged when we start using `4.13.x` bundles.

### CODE REVIEW

**Comment by jsliacan** [comment]
/hold

Need to wait until CI uses 4.13 bundles.

**Review by gbraad — APPROVED** [review]

**Comment by gbraad** [comment]
> /home

Why? For use with OCP 4.13 only... if so, please specify this reason

**Comment by gbraad** [comment]
Accidental close... ugh...

**Review by adrianriobo — APPROVED** [review]

**Comment by praveenkumar** [comment]
@jsliacan Can you rebase it once #3680 merge so that this test is also get verified before we merge this PR.

**Comment by praveenkumar** [comment]
As per https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/crc-org_crc/3680/pull-ci-crc-org-crc-main-e2e-crc/1662020038300798976 looks like CI only failing because of this, so I am merging this one

**Pull request merged**
